arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/mmu/opcode/Create2.java

```diff
@@ -45,6 +45,9 @@ public Create2(final Hub hub, final boolean failedCreate) {
                 hub.currentFrame()
                     .frame()
                     .shadowReadMemory(0, hub.currentFrame().frame().memoryByteSize())))
+        .exoBytes(
+            Optional.of(
+                hub.romLex().getCodeByMetadata(contract))) // TODO ? What about for failed CREATE2 ?
```

Comment:
For the failed CREATE2 we still need to send stuff to SHAKIRA. But not to ROM.

Reply:
I'm not sure how this works here but in any case you will need to activate both SHAKIRA and (potentially) ROM.

Reply:
yes, you're absolutely right. Th